### PR TITLE
CurrentUser API UG documentation

### PR DESCRIPTION
**Pull request by YathavakrishnanMohan**
Description :
Add the UG documentation for CurrentUser API

### CODE REVIEW

File: wpf-toc.html [hunk 1]
```diff
@@ -1528,6 +1528,7 @@
 					<li><a href="/wpf/Pdf-Viewer/how-to/disable-toolbar-items">Disable toolbar items</a></li>
 					<li><a href="/wpf/Pdf-Viewer/how-to/disable-the-annotations-selection">Disable the annotations selection</a></li>
 					<li><a href="/wpf/Pdf-Viewer/acquiring-total-number-of-pages">Acquire total number of pages</a></li>
+					<li><a href="/wpf/Pdf-Viewer/Changing-the-Current-User">Change the CurrentUser of the annotations</a></li>
```
Comment (RajaVigneshB) [inline wpf-toc.html:1531]: Please move the new md file inside the How-To folder

File: wpf/Pdf-Viewer/Changing-the-Current-User.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
+---
+layout: post
+title: Change the CurrentUser of annotations in Pdf Viewer | Syncfusion
+description: Learn about Change the CurrentUser of annotations in Syncfusion WPF Pdf Viewer control.
```
Comment (RajaVigneshB) [inline wpf/Pdf-Viewer/Changing-the-Current-User.md:4]: Description was not proper. Change it

**Review by RajaVigneshB — CHANGES_REQUESTED** [review]

File: wpf-toc.html (hunk 1 above)
Comment (YathavakrishnanMohan) [reply wpf-toc.html]: File moved to How-To folder is completed

File: wpf/Pdf-Viewer/Changing-the-Current-User.md (hunk 2 above)
Comment (YathavakrishnanMohan) [reply wpf/Pdf-Viewer/Changing-the-Current-User.md]: Description is changed properly

File: wpf-toc.html [hunk 3]
```diff
@@ -1528,6 +1528,7 @@
 					<li><a href="/wpf/Pdf-Viewer/how-to/disable-toolbar-items">Disable toolbar items</a></li>
 					<li><a href="/wpf/Pdf-Viewer/how-to/disable-the-annotations-selection">Disable the annotations selection</a></li>
 					<li><a href="/wpf/Pdf-Viewer/acquiring-total-number-of-pages">Acquire total number of pages</a></li>
+					<li><a href="/wpf/Pdf-Viewer/how-to/Changing-the-Current-User">Change the CurrentUser of the annotations</a></li>
```
Comment (Vikassekar) [inline wpf-toc.html:1531]: Current User is not only for the annotation so you can "change it as Change the current user in PdfViewer

File: wpf/Pdf-Viewer/How-To/Changing-the-Current-User.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
+---
+layout: post
+title: Change the CurrentUser of annotations in Pdf Viewer | Syncfusion
```
Comment (Vikassekar) [inline wpf/Pdf-Viewer/How-To/Changing-the-Current-User.md:3]: modify the title and describtion.

File: wpf/Pdf-Viewer/How-To/Changing-the-Current-User.md [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+
+# Change the CurrentUser of annotations in WPF Pdf Viewer
+
+PDF Viewer allows you to change the CurrentUser of annotations. The following code example illustrates the same.
+
+{% tabs %}
+{% highlight C# %}
+
+//Initialize PDF Viewer.
+PdfViewerControl pdfViewer1 = new PdfViewerControl();
```
Comment (Vikassekar) [inline wpf/Pdf-Viewer/How-To/Changing-the-Current-User.md:18]: Provide proper naming

**Review by Vikassekar — CHANGES_REQUESTED** [review]

File: wpf-toc.html (hunk 3 above)
Comment (YathavakrishnanMohan) [reply wpf-toc.html:1531]: Title is changed to  "Change the current user in PdfViewer" completed

File: wpf/Pdf-Viewer/How-To/Changing-the-Current-User.md (hunk 4 above)
Comment (YathavakrishnanMohan) [reply wpf/Pdf-Viewer/How-To/Changing-the-Current-User.md:3]: title and description is modified

File: wpf/Pdf-Viewer/How-To/Changing-the-Current-User.md (hunk 5 above)
Comment (YathavakrishnanMohan) [reply wpf/Pdf-Viewer/How-To/Changing-the-Current-User.md:18]: Naming is changed proper naming

File: wpf/Pdf-Viewer/How-To/Changing-the-Current-User.md [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+title: Change the Current user in PdfViewer | Syncfusion
+description: Learn about how to Change the CurrentUser in Syncfusion WPF Pdf Viewer control using CurrentUser property.
+platform: wpf
+control: PDF Viewer
+documentation: ug
+---
+
+# Change the CurrentUser in WPF Pdf Viewer
+
+The PDF Viewer allows you to change the CurrentUser. If the CurrentUser property is not set, it defaults to the system user name. When you set the CurrentUser, you will see the changes in the annotation properties when adding an annotation to the document. The following code example illustrates this:
```
Comment (Vikassekar) [inline wpf/Pdf-Viewer/How-To/Changing-the-Current-User.md:12]: modify the content as below 
The PDF Viewer allows you to change the CurrentUser. If the CurrentUser property is not set, it defaults to the system user name. When you set the CurrentUser, the changes will be reflected in the author property of newly added annotations. The following code example illustrates how to set the CurrentUser:

**Review by Vikassekar — CHANGES_REQUESTED** [review]

File: wpf/Pdf-Viewer/How-To/Changing-the-Current-User.md (hunk 6 above)
Comment (YathavakrishnanMohan) [reply wpf/Pdf-Viewer/How-To/Changing-the-Current-User.md:12]: Content changing is completed

**Review by Vikassekar — APPROVED** [review]

**Review by Vikassekar — APPROVED** [review]

**Review by RajaVigneshB — APPROVED** [review]

**Pull request merged**